### ISSUE TITLE
[node] Add init system

### ISSUE THREAD

**Issue by eddiezane** [chainguard-images/images#271]
Node.js was not designed to run as PID 1 and doing so causes headaches.

We should explore adding an init system of some sort -- the node docs recommend tini.

https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md#handling-kernel-signals

https://github.com/krallin/tini#using-tini

**Comment by eddiezane** [comment]
Slack discussion suggests we use the apko s6 builtin service for this.

https://chainguard-dev.slack.com/archives/C0322APD4LU/p1676331584605819

**Comment by patflynn** [comment]
@amouat does moving this image to use the same s6 service make sense to prioritze?  I think the image is slightly broken as is. https://github.com/chainguard-images/images/blob/main/images/nginx/configs/latest.apko.yaml

**Comment by amouat** [comment]
Yes, it probably does. I remember the Docker Node images never respond to signals etc

**Comment by amouat** [comment]
I did have a brief look at this before and I'm not sure how to solve it. The issue is that the base image doesn't really need to start with an init system, it's the images that are built on top of this. Maybe we could create a clever entrypoint or some best practices on how to achieve this?

**Comment by amouat** [comment]
I don't think I can do much with this at this time, but it is something to be aware of and improve if we can.
